src/validators/mod.rs

```diff
@@ -58,6 +58,25 @@ pub use with_default::DefaultType;
 
 use self::definitions::DefinitionRefValidator;
 
+#[pyclass(module = "pydantic_core._pydantic_core", name = "Some")]
+pub struct PySome {
+    #[pyo3(get)]
+    value: PyObject,
+}
```

Comment:
We need a way to differentiate "no default value" from `None is the default value`.
The other option would be to make it an error, but it can't (currently) be a `ValidationError` because `ValError` requires an `Input` which we don't have here. I also think there's something to be said for differentiating errors stemming from the default value failing validation from the absence of a default value. So maybe a `LookupError` would work if we decided to go the error route.